contents/uix/accessibility.md

```diff
@@ -0,0 +1,181 @@
+![Facebook sign up page to someone with deuteranopia](accessibility/fb-deuteranopia.png "Facebook sign up page to someone with deuteranopia")
+
+
+It may be unclear to color-blind users that the sign up has failed. To avoid confusion, we should not rely on colours alone to convey a message. Instead, we should use more visual indicators such as icons or an explanation box.
+![Sign up page with more visual indicators](accessibility/fb-more-indicators.png "Facebook sign up page with more visual indicators")
+
+##### :fas-thumbs-up: Best practice: Maintain good contrast
+  
+Look at this page taken from Tech Crunch.
+![Tech Crunch page](accessibility/tc-normal.png "Tech Crunch page")
+
+To users with good vision, the design may look minimalist and clean. However, to someone suffering from cataract, the page looks like this.
+
+![Blurry Tech Crunch page](accessibility/tc-blurred.png "Tech Crunch page to someone with cataract")
+It gets difficult to read the news snippets because of the poor color contrast (grey on white). On the other hand, the black colored texts are still legible. Thus, we should maintain good contrast ratio in our websites for ease of reading.
+
+##### :fas-thumbs-up: Best practice: Use `alt` attribute for images
+  
+The `alt` attribute provides alternative information for an image that can be read out by a screen reader to describe an image.
+
+![Use of alt tag](accessibility/alt-tag.png "Alt tag")
+
+The above are some examples of best practices to follow when developing for web accessibility. More can be found in [this article on developing for accessibility](https://www.w3.org/WAI/tips/developing/).
+  
+### Auditory
+Hearing-impaired users do use assistive technologies such as a hearing aid or a cochlear implant, but these are not specific for accessing websites.
+
+For users with hearing impairment, we should provide text alternatives to audio content, such as:
+  * Text transcripts
+  * Captions
+
+Here's a [tutorial on adding transcripts](https://www.w3.org/2017/04/xsmedia-tute/page5.html). Adding captions is commonly supported in video hosting websites such as Youtube or Vimeo.
+
+More information can be found on [this article on transcripts and captions](https://webaim.org/techniques/captions/).
+  
+### Cognitive
+Cognitive disabilities range from mental illnesses to learning difficulties, difficulties in comprehension and concentration, etc. Some examples include [Attention Deficit Hyperactivity Disorder (ADHD)](https://en.wikipedia.org/wiki/Attention_deficit_hyperactivity_disorder), and [autism](https://en.wikipedia.org/wiki/Autism).
+
+Such disabilities might affect website usage. It could be due to difficulty in understanding how to complete a task, remembering how to do something that was previously accomplished, increased frustration at confusing workflows or inconsistent layouts/navigation/other page features.
```

Comment:
> Such disabilities might affect website usage.

I think rephrasing it to `Such disabilities affect how the user uses a website` would be better. 

The original sentence sounds like having a disability affects how "other" users use the website. For example, "Having a slow network affects website usage".

> increased frustration at confusing workflows

increased frustration due to confusing workflows